Dynatrace/dynatrace-operator
# Script for generating permission tables for documentation

## Description

Small python script that takes an operator manifest as input and writes markdown tables with all permissions required by `Roles` and `ClusterRoles`. The resulting markdown can be used to request updates of our official documentation.

## How can this be tested?

* Make sure that you have the required pre-requisites installed on your machine (hack/doc/README.md)
* run make doc/permissions

## Checklist

- [n/a] Unit tests have been updated/added
